### PR TITLE
Save First and Last Name while Creating a user account

### PR DESCRIPTION
**Pull request by arafatkn**
Google account' first and last name will be saved to WP user account.

It will Fix [This Issue #101](https://github.com/rtCamp/login-with-google/issues/101)

### CODE REVIEW

File: src/Utils/Authenticator.php [hunk 1]
```diff
@@ -95,6 +95,8 @@ public function register( stdClass $user ): ?WP_User {
 						'user_login' => Helper::unique_username( $user->login ),
 						'user_pass'  => wp_generate_password( 18 ),
 						'user_email' => $user->email,
+						'first_name' => $user->given_name,
+						'last_name'  => $user->family_name,
```
Comment (ankitrox) [inline src/Utils/Authenticator.php:99]:
```suggestion
						'first_name' => $user->given_name ?? '',
						'last_name'  => $user->family_name ?? '',
```

File: tests/php/Unit/Utils/AuthenticatorTest.php [hunk 2]
```diff
@@ -213,6 +215,8 @@ public function testRegisterReturnsUserObject() {
 					'user_login' => 'test',
 					'user_pass'  => 'thisisrandompass',
 					'user_email' => 'test@example.com',
+					'first_name' => 'Test',
+					'last_name'  => 'User',
```
Comment (ankitrox) [inline tests/php/Unit/Utils/AuthenticatorTest.php:219]: Add a test where name and last name both are empty. Also another test where one of these will be empty.

**Review by ankitrox — CHANGES_REQUESTED** [review]

**Review by ankitrox — APPROVED** [review]
